packages/lesswrong/server/languageModels/modGPT.ts

```diff
@@ -204,6 +205,16 @@ async function checkModGPT(comment: DbComment): Promise<void> {
         currentUser: adminsAccount,
         validate: false
       })
+      
+      // also add a note for mods
+      await updateMutator({
+        collection: Users,
+        documentId: comment.userId,
+        set: {
+          sunshineNotes: getSignatureWithNote('ModGPT', `Intervened on comment ID=${comment._id}`) + user.sunshineNotes
+        },
+        validate: false
+      })
```

Comment:
Once again I am disappointed by the moderation system. You should be adding a moderator action and specifying flag: false, triggerReview: false, IMO. But there's no way to do that.